server-manager.proto

```diff
@@ -275,9 +250,8 @@ message FinishEpochRequest {
 
 message FinishEpochResponse {
     CartesiMachine.Hash machine_hash = 1; // Machine hash in epoch
-    CartesiMachine.Hash vouchers_epoch_root_hash = 2; // Root hash for Merkle tree of voucher hashes memory ranges
-    CartesiMachine.Hash notices_epoch_root_hash = 3; // Root hash for Merkle tree of notice hashes memory ranges
-    repeated Proof proofs = 4; // Proofs for the outputs
+    CartesiMachine.Hash outputs_epoch_root_hash = 2; // Merkle root of all outputs_epoch_root_hash for inputs in epoch
```

Comment:
Did somebody ask for this repeated field?

Reply:
We need this field in the node even if len(proofs) == 0